src/utils/viewer/viewer.cpp

```diff
@@ -445,7 +445,7 @@ void Viewer::toggleNavMeshVisualization() {
     int nevMeshVisPrimID = resourceManager_.loadNavMeshVisualization(
         *pathfinder_, navmeshVisNode_, &sceneGraph_->getDrawables());
     navmeshVisNode_->translate({0, 0.1, 0});
-  } else {
+  } else if (navmeshVisNode_ != nullptr) {
```

Comment:
Not that it matters much in this case (no followup PR needed), but `delete` (and `free()` as well) is a no-op when called on a null pointer, so this additional check isn't strictly needed. 

What makes the fix work is initializing the variable to `nullptr` above.